src/client/pages/OfferNew/Introduction/Sidebar/DetailsModal/Details/components/Details.styles.tsx

```diff
@@ -0,0 +1,63 @@
+import styled from '@emotion/styled'
```

Comment:
In general i agree here! But - since these styles are specific enough for the details form, but they cannot be placed in the `index.tsx` file (since the dependencies would become circular if the files imported from `index.tsx` depended on `index.tsx`) so i see the reasoning for placing them in a special file for this use case. I like this naming more than the previous, but maybe we could work on it more 😅  maybe `formStyles`  or something? Imo it's encapuslated as "the component of Details" and not hard to make out that these styles belong with the sibling files. It'd be en annan femma if it was cross reference to a different component or what do you think?